components/apimgt/org.wso2.carbon.apimgt.rest.api.devops/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ /*
+  ~  * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~  *
+  ~  * WSO2 Inc. licenses this file to you under the Apache License,
+  ~  * Version 2.0 (the "License"); you may not use this file except
+  ~  * in compliance with the License.
+  ~  * You may obtain a copy of the License at
+  ~  *
+  ~  * http://www.apache.org/licenses/LICENSE-2.0
+  ~  *
+  ~  * Unless required by applicable law or agreed to in writing,
+  ~  * software distributed under the License is distributed on an
+  ~  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~  * KIND, either express or implied. See the License for the
+  ~  * specific language governing permissions and limitations
+  ~  * under the License.
+  ~  */
+  -->
+<web-app xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://java.sun.com/xml/ns/javaee"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
+         version="3.0" metadata-complete="true">
+    <absolute-ordering />
+
+    <description>JAX-WS/JAX-RS Device Registration Agent Endpoint</description>
+    <display-name>JAX-WS/JAX-RS Servlet</display-name>
+    <context-param>
+        <param-name>contextConfigLocation</param-name>
+        <param-value>WEB-INF/beans.xml</param-value>
+    </context-param>
+    <listener>
+        <listener-class>
+            org.springframework.web.context.ContextLoaderListener
+        </listener-class>
+    </listener>
+    <servlet>
+        <description>JAX-WS/JAX-RS Device Registration Agent Endpoint</description>
```

Comment:
change the description

Reply:
changed the display name and description in line 27-28 and removed this line adhering to web.xml files in publisher/store/gateway rest apis.